BLUEPRINT/systems/plotting.py

```diff
@@ -184,7 +184,7 @@ def plot_1D(self, width=1.0):
         # TODO: POSTprocess.py is archaic... need to refactor heavily and bring
         # into the fold...
         sys_code_dir = self.reactor.file_manager.generated_data_dirs["systems_code"]
-        codes.plot_PROCESS(os.sep.join([sys_code_dir, "OUT.DAT"]), width=width)
+        codes.plot_radial_build(os.sep.join([sys_code_dir, "OUT.DAT"]), width=width)
```

Comment:
This output filename is specific to PROCESS. If this was just in BLUEPRINT we could leave as is. I do wonder how we'd plot the PROCESS output in the EUDEMO reactor though. Not sure how best to handle this, but another systems code will likely how a different radial build plotting method. The one we have for PROCESS right now is really janky because it works on the OUT.DAT directly.

My instinct is always to add a plot method to a solver... Maybe that is bad